src/sota_tools/authenticate.cc

```diff
@@ -131,6 +143,10 @@ int authenticate(const string &cacerts, string filepath, TreehubServer &treehub)
       }
       break;
     }
+    case CERT: {
+      treehub.SetCerts(root_cert, client_cert, client_key);
```

Comment:
Is it worth checking at this point that all three cert strings are non-empty? I think it's better to fail early and explicitly in that case.

Reply:
Ok

Reply:
Looks good, thanks.